packages/components/tabs/src/tab-group.ts

```diff
@@ -83,11 +83,11 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
    */
   #mutationObserver = new MutationObserver(entries => {
     entries.forEach(entry => {
-      if (entry.attributeName === 'selected' && entry.oldValue === null) {
+      if (entry.attributeName === 'selected' && entry.oldValue === null && entry.target instanceof Tab) {
```

Comment:
Why is this needed?

Reply:
There was an issue, when on the website `sl-select` was added (in the sl-tab-panel) which also has `selected` attribute, we need to take only tab into account here.